pulpcore/cli/core/context.py

```diff
@@ -262,6 +263,17 @@ def scope(self) -> Dict[str, Any]:
         else:
             return {}
 
+    def purge(self, finished_before: Optional[str], states: Optional[List[str]]) -> Any:
+        body: Dict[str, Union[str, List[str]]] = {}
```

Comment:
I appreciate the effort, but for these, we usually do `body: Dict[str, Any]`, because `Any` is the closest thing we have to `JSONSerializable`, or `ConstructOfPythonPrimitives`.